playbooks/common/openshift-cluster/upgrades/post_control_plane.yml

```diff
@@ -66,6 +69,9 @@
       {{ oc_cmd }} patch dc/docker-registry -n default -p
       '{"spec":{"template":{"spec":{"containers":[{"name":"registry","image":"{{ registry_image }}"}]}}}}'
       --api-version=v1
+    # AUDIT:changed_when_note: `false` not being set here. What we
```

Comment:
I think @tbielawa is noting this is an area that needs to be changed and, for now, it is technically a ``changed_when: true`` due to the ``patch``.